### PR TITLE
Error in notification

### PR DESCRIPTION
**Pull request by DCzajkowski**
Moves all QWErrors into notifications. Previous behaviour: pasted errors as if they were a replacement string

### CODE REVIEW

File: src/main/modules/SnippetsManager.js [hunk 1]
```diff
@@ -195,7 +196,13 @@ class SnippetsManager {
...
             return await this._evaluate(matchedString, code)
         } catch (error) {
-            return `QWError: ${_.get('error', 'message', String(error))}`
+            if (!Notification.isSupported()) {
+                return `QWError ${_.get('error', 'message', String(error))}`
+            }
+
+            Notification.show('QWError', _.get('error', 'message', String(error)))
+
+            return ''
```
Comment (gtluszcz) [inline src/main/modules/SnippetsManager.js:205]: Czy chcemy podmienić na pusty string czy na klucz ?
Jakie zachowanie byłoby bardziej intuicyjne?

EDIT by @DCzajkowski: English translation: "Do we want to substitute for an empty string or for a trigger? Which behaviour is more intuitive?"

**Review by gtluszcz — APPROVED** [review]
Zajebista robota, świetny pomysł 👍

File: src/main/modules/SnippetsManager.js (hunk 1 above)
Comment (DCzajkowski) [reply src/main/modules/SnippetsManager.js:205]: I think substitution for a trigger may be misleading in a way, people would like to re-trigger the shortcut. Imagine you type `;abcd`, a notification pops up and `;abcd` stays. People may try to remove the last letter and retype it, but it'd have no effect. What do you think?

**Comment by DCzajkowski** [comment]
Going to merge this for now, but I am still open to discussing what should it be replaced to ;)

**Pull request merged**